persistence/src/main/java/org/mskcc/cmo/metadb/persistence/neo4j/MetadbSampleRepository.java

```diff
@@ -83,4 +83,8 @@ MetadbSample findResearchSampleByRequestAndIgoId(@Param("reqId") String reqId,
             + "RETURN sm")
     List<SampleMetadata> findSampleMetadataHistoryByNamespaceValue(
             @Param("namespace") String namespace, @Param("value") String value);
+
+    @Query("MATCH (r: Request)-[:HAS_SAMPLE]->(s: Sample {metaDbSampleId: $metaDbSampleId}) "
+            + "RETURN r.isCmoRequest")
+    String findSampleHasCmoRequest(@Param("metaDbSampleId") UUID metaDbSampleId);
```

Comment:
Query to check if the request of a sample is cmo or non-cmo

Reply:
[requested change] expanding `PublishedMetadbSample` with the `SampleMetadata.additionalProperties` will allow us to avoid having to need this query

Reply:
This query is used when we're setting up new `publishedMetadbSamples` to populate `isCmoSample`

Reply:
I'm still hesitant to keep this query but it's fine for now. 

can you rename to `findSampleCmoRequestStatus`